cloudformation/jupiterone-cloudformation.json

```diff
@@ -46,14 +46,14 @@
                 "AWS": {
                   "Fn::Join": [
                     "",
-                    ["arn:aws:iam::", { "Ref": "J1AccountID" }, ":root"]
+                    ["arn:aws:iam::", { "Ref": "J1AwsAccountId" }, ":role/JupiterIntegrationAws*"]
```

Comment:
Rather than allowing the entire account the ability to assume role, we scope it to the j1 integration aws roles. I added the asterisk (`*`) because we will end up having multiple workers, each with their own role, assuming this one.